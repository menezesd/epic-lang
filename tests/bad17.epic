func main() {
    // anything before runtime error must be printed
    print 1;
    print 2;
    print 3;
    // here is the error ;(
    if [1] then {}
}