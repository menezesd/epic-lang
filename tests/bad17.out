1
2
3
runtime error
